Dockerfile

```diff
@@ -3,6 +3,6 @@
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY manager .
-USER nonroot:nonroot
+USER 65532:65532
```

Comment:
Please revert this change: https://github.com/goharbor/harbor-operator/pull/420#discussion_r569434764